pkg/commands/initialize/init_test.go

```diff
@@ -175,23 +201,28 @@ func TestInitProcess(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
 
-			githubClient := &github.MockGitHubClient{}
+			actions := githubactions.New(githubactions.WithWriter(os.Stdout))
```

Comment:
I don't think you want to write to stdout in the test. If you don't need the output, you can use `io.Discard`